packages/cli/src/__tests__/commands/DebugInfo.test.ts

```diff
@@ -55,19 +55,17 @@ const envVars = {
 }
 
 describe('debug', () => {
-  beforeEach(() => {
-    process.env = { ...originalEnv, ...envVars }
-  })
-  afterAll(() => {
+  // clean up env vars after each individual test
```

Comment:
This may be useful to ensure no other env vars leak
```suggestion
  // make sure the env is empty before each test
  beforeEach(() => {
    process.env = {}
  })
  // clean up env vars after each individual test
```